java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -49,16 +44,14 @@ public void startSession_defaultConfig_usWestDataCenter() {
 
     @Test
     public void getUserName_usernameSetInEnvironmentVariable_returnsValue() {
-        when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_USERNAME")).thenReturn("test-name");
-        String actualUserName = sauce.getUserName();
-        assertNotEquals("",actualUserName);
+        String actualUserName = sauce.getSauceUsername();
+        assertNotEquals("test-name",actualUserName);
     }
 
     @Test
     public void getAccessKey_keySetInEnvironmentVariable_returnsValue() {
-        when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_ACCESS_KEY")).thenReturn("accessKey");
-        String actualAccessKey = sauce.getAccessKey();
-        assertNotEquals("", actualAccessKey);
+        String actualAccessKey = sauce.getSauceAccessKey();
```

Comment:
Same thing here, I'd expect it to return "accesskey" and **be equal**

Reply:
yeah, it's taking advantage of the fact that my "real" `ENV["SAUCE_ACCESS_KEY"]` is not "accesskey" which I agree is kind of silly test.